processor/deltatocumulativeprocessor/internal/telemetry/metrics.go

```diff
@@ -0,0 +1,199 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package telemetry // import "github.com/open-telemetry/opentelemetry-collector-contrib/processor/deltatocumulativeprocessor/internal/telemetry"
```

Comment:
Given that this package defines not only the metrics, but also some behavior, this needs tests. Documentation would also be good to have.

Reply:
added error handling behavior tests